app/src/main/java/com/naman14/timberx/util/NotificationUtils.kt

```diff
@@ -111,32 +110,30 @@ object NotificationUtils {
                         PendingIntent.getService(context, 0, actionIntent.apply { action = Constants.ACTION_NEXT }, 0)))
 
         if (artwork != null) {
-            builder.setColor(Palette.from(artwork).generate().getVibrantColor(Color.parseColor("#403f4d")))
+            builder.color = Palette.from(artwork).generate().getVibrantColor(Color.parseColor("#403f4d"))
         }
 
-        val n = builder.build()
+        return builder.build()
+    }
 
-        return n
+    fun updateNotification(context: Context, mediaSession: MediaSessionCompat) {
+        doAsync {
+            (context.getSystemService(Context.NOTIFICATION_SERVICE) as NotificationManager)
```

Comment:
I was thinking of https://github.com/android/android-ktx/blob/136ba4cdb3b6ece7470cbddeaf6a168021a69a30/src/main/java/androidx/core/content/Context.kt#L37 but was hoping there is a `val Context.notificationManager get() = getSystemService(Context.NOTIFICATION_SERVICE) as NotificationManager`

Reply:
@Zhuinden dependency injection will clean this up later